client/src/components/Button/Button.css

```diff
@@ -15,47 +15,47 @@
 .button:hover {
   background-color: #fcf0ee;
 }
-.button-large {
+.button--large {
```

Comment:
This looks great! Can we change all the bs to be uppercase for consistency?

.Button--large